transform.py

```diff
@@ -1 +1,121 @@
+from datetime import datetime
```

Comment:
create base classes for : extract, transform . per data source you should create a class that inheritance from the parent class and implement this source extractor. you should have one loader because all transformers should translate data to the same data structure to upload